packages/ui-toolkit/src/boxes/index.js

```diff
@@ -11,19 +11,9 @@ export const tooltipShadow = `0 ${remcalc(2)} ${remcalc(6)} ${remcalc(
 export const modalShadow = `0 0 ${remcalc(6)} ${remcalc(1)} rgba(0, 0, 0, 0.1)`;
 
 export const border = {
-  checked: css`
-    ${remcalc(1)} solid ${props => props.theme.primary};
-  `,
-  unchecked: css`
-    ${remcalc(1)} solid ${props => props.theme.grey};
-  `,
-  confirmed: css`
-    ${remcalc(1)} solid ${props => props.theme.grey};
-  `,
-  error: css`
-    ${remcalc(1)} solid ${props => props.theme.red};
-  `,
-  secondary: css`
-    ${remcalc(1)} solid ${props => props.theme.secondaryActive};
-  `
+  checked: css`${remcalc(1)} solid ${props => props.theme.primary};`,
```

Comment:
prettier :(

Reply:
I think the unreleased version of prettier handles that better